crates/miden-objects/src/note/recipient.rs

```diff
@@ -24,17 +24,17 @@ use super::{
 ///
 /// Recipient is computed as:
 ///
-/// > hash(hash(hash(serial_num, [0; 4]), script_root), input_commitment)
+/// > hash(hash(hash(serial_num, [0; 4]), script_root), storage_commitment)
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct NoteRecipient {
     serial_num: Word,
     script: NoteScript,
-    inputs: NoteInputs,
+    inputs: NoteStorage,
     digest: Word,
 }
 
 impl NoteRecipient {
-    pub fn new(serial_num: Word, script: NoteScript, inputs: NoteInputs) -> Self {
+    pub fn new(serial_num: Word, script: NoteScript, inputs: NoteStorage) -> Self {
```

Comment:
```suggestion
    pub fn new(serial_num: Word, script: NoteScript, storage: NoteStorage) -> Self {
```

All of the variable names should also be updated (applies to the entire PR),